src/SprykerEco/Client/Computop/ComputopQuoteDefaultShipmentExpander.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Client\Computop;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use SprykerEco\Client\Computop\Zed\ComputopStubInterface;
+
+class ComputopQuoteDefaultShipmentExpander implements ComputopQuoteDefaultShipmentExpanderInterface
+{
+    /**
+     * @var \SprykerEco\Client\Computop\Zed\ComputopStubInterface $computopStub
+     */
+    protected $computopStub;
+
+    /**
+     * @param \SprykerEco\Client\Computop\Zed\ComputopStubInterface $computopStub
+     */
+    public function __construct(ComputopStubInterface $computopStub)
+    {
+        $this->computopStub = $computopStub;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function expandQuoteWithDefaultShippingMethod(QuoteTransfer $quoteTransfer): QuoteTransfer
+    {
+        if ($quoteTransfer->getItems()->count() === 0 || $this->isQuoteHasShipment($quoteTransfer)) {
```

Comment:
Shouldn't we also check Quote-level shipment?

Reply:
Yes, it is empty in the latest suite, but some customers might still have it, that's why we should support both item-level and deprecated quote-level shipment.

Reply:
Fixed